action.yml

```diff
@@ -80,12 +80,20 @@ inputs:
       Please refer to this action's README for all available placeholders.
     default: >-
       [Backport ${target_branch}] ${pull_title}
+  pull_branchname:
+    description: ""
+    default: >-
+      backport-${pull_number}-to-${target}
   target_branches:
     description: >
       The action will backport the pull request to each specified target branch (space-delimited).
       Note that the pull request's headref is excluded automatically.
       Can be used in addition to backport labels.
       By default, only backport labels are used to specify the target branches.
+  target_owner:
+    description: ""
+  target_repo:
+    description: ""
```

Comment:
👍 I like the flexibility and naming. 

🔧 For some additional suggestions, we could consider `remote_owner` and `remote_repo` as valid options. These might overcome the potential naming confusion mentioned [here](https://github.com/korthout/backport-action/issues/384#issuecomment-1715746551). When they are not specified, the current working repo is used. Using `remote` in the naming clarifies what this really does: backport to a remote repo.

💭 We could also consider a single `target_repo` input that takes the full repo identifier `owner/name`. This would allow setting the default using the [github context](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context) to `${{ github.repository }}`. Sadly the github context does not have the repo name only. A downside to this approach is that we must parse the string to separate the owner and repo for later usage. Additionally, if we have some logic specific to the remote target, then we must compare the value to the context's value to see if it was explicitly defined. So, I'm leaning away from the single `target_repo` input approach.

Reply:
Good point 👍 Changed it accordingly.